src/ReferenceList.php

```diff
@@ -24,6 +24,9 @@
  * @author H. Snater < mediawiki@snater.com >
  * @author Thiemo Kreuz
  * @author Bene* < benestar.wikimedia@gmail.com >
+ *
+ * @phpcs:disable MediaWiki.NamingConventions.LowerCamelFunctionsName.FunctionName
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
```

Comment:
How would you feel about altering the mess detector configuration to also add __ to the ignorepattern for the TooManyPublicMethods rule?
I think that might make sense, and then this suppression could also be removed.

Reply:
Yes i can try adding this now!